android/app/build.gradle

```diff
@@ -266,15 +267,15 @@ allprojects {
     repositories {
         configurations.all {
             // To fix the app start crash error
-            resolutionStrategy.eachDependency { DependencyResolveDetails details ->
-                def requested = details.requested
-                if (requested.group == 'com.google.android.gms') {
-                    details.useVersion '12.0.1'
-                }
-                if (requested.group == 'com.google.firebase') {
-                    details.useVersion '12.0.1'
-                }
-            }
+            // resolutionStrategy.eachDependency { DependencyResolveDetails details ->
+            //     def requested = details.requested
+            //     if (requested.group == 'com.google.android.gms') {
+            //         details.useVersion '12.0.1'
+            //     }
+            //     if (requested.group == 'com.google.firebase') {
+            //         details.useVersion '12.0.1'
+            //     }
+            // }
```

Comment:
Why leave these codes commented and not delete them? or are they going to be used in the future?

Reply:
This pease of code were establishing the firebase and google version in the app, I commented out because I used firebase-BoM instead, but this code is from MIT and I do not know either  we'll use it again or we'll want to go back to this configuration so I left it